modules/core/vault-helper.sh

```diff
@@ -14,7 +14,7 @@ readonly SCRIPT_NAME="$(basename "$0")"
 readonly MAX_RETRIES=30
 readonly SLEEP_BETWEEN_RETRIES_SEC=10
 
-TERRAFORM="${TERRAFORM:-terraform}"
+TERRAFORM="${TERRAFORM:-terragrunt}"
```

Comment:
Let's not change the default to `terragrunt`. We might use that internally, but it's not always the default.

You can override the default when running the script by using `TERRAFORM=terragrunt vault-helper.sh`